src/firebolt/service/V2/engine.py

```diff
@@ -129,11 +164,35 @@ def _format_engine_parameter(
             return str(value.value)
         return value
 
+    def _validate_create_parameters(
+        self,
+        **create_parameters: Union[
+            str, int, EngineType, InstanceType, WarmupMethod, None
+        ],
+    ) -> None:
+        if self.client._account_version == 2:
+            bad_parameters = [
+                name
+                for name in self.DISALLOWED_ACCOUNT_V2_PARAMETERS
+                if create_parameters.get(name) is not None
+            ]
+            if bad_parameters:
+                raise ValueError(
+                    f"Parameters {bad_parameters} are not supported for this account"
+                )
+
+    def _format_engine_attribute_sql(
+        self, param: str, value: Union[str, int, EngineType, InstanceType, WarmupMethod]
+    ) -> Tuple[str, List]:
+        if param == "TYPE":
+            return f"{param} = {self._format_engine_parameter(value)} ", []
+        return f"{param} = ? ", [self._format_engine_parameter(value)]
```

Comment:
Maybe let's pop a comment on what this method returns. Otherwise it's not clear what the List is supposed to be.